### ISSUE TITLE
Enter submits form in Firefox

### ISSUE THREAD

**Issue by drweissbrot** [jshjohnson/Choices#462]
I'm using a multiselect instance of Choices, within a `<form>`. When I select a choice by pressing Enter, Firefox will automatically try and submit the form.

An example in a Codepen: https://codepen.io/drweissbrot/full/QJGqGj/

The console mentions `ReferenceError: event is not defined` in https://github.com/jshjohnson/Choices/blob/master/src/scripts/choices.js#L1258.

Everything is working properly when clicking to select a choice.

This appears to only be a problem in Firefox, everything is working fine in Chrome. I'm using Choices 4.1.1 beta, the problem also persists in 4.1.0.

**Comment by atomgiant** [comment]
I am seeing the same thing.  As a temporary work around I'm using this jQuery to disable all form submit with Enter

```javascript
    $(document).on("keypress", ":input:not(textarea)", function(event) {
      return event.keyCode != 13;
    });
```

**Comment by drweissbrot** [comment]
Thanks! Had to make some changes since I don't use jQuery, but it seems to work.
Obviously not a permanent solution though :/

**Comment by jshjohnson** [comment]
Yeah this is a bug - will investigate 👍

**Comment by jshjohnson** [comment]
This has been resolved in [version 4.1.2](https://github.com/jshjohnson/Choices/releases/tag/v4.1.2). Thanks 👍
